src/apps/activity_assignments/router.py

```diff
@@ -22,6 +27,21 @@
     },
 )(assignments_create)
 
+router.delete(
+    "/applet/{applet_id}/unassigns",
+    description="""Unassign a set of activity assignments. For each
```

Comment:
if use http DELETE, maybe we should use "/applet/{applet_id}/assignments", because we are deleting "assignments" not "unassigns", but if you want to keep path, maybe should change to POST

Reply:
You are right @farmerpaul I missed the prefix "/assignments"

Reply:
> You are right @farmerpaul I missed the prefix "/assignments"

This works. And if ever needed down the road, this still leave room for deleting a single assignment by ID with a path like DELETE /assignments/{assignment_id}

Reply:
changes post to delete and keep same route as assignment